src/components/notifications/Twilio.vue

```diff
@@ -1,32 +1,36 @@
 <template>
     <div class="mb-3">
-        <label for="twilio-account-sid" class="form-label">{{ $t("Account SID") }}</label>
+        <label for="twilio-account-sid" class="form-label">{{ $t("twilioAccountSID") }}</label>
         <input id="twilio-account-sid" v-model="$parent.notification.twilioAccountSID" type="text" class="form-control" required>
     </div>
 
     <div class="mb-3">
-        <label for="twilio-apikey-token" class="form-label">{{ $t("Api Key (optional)") }}</label>
+        <label for="twilio-apikey-token" class="form-label">{{ $t("twilioApiKey") }}</label>
         <input id="twilio-apikey-token" v-model="$parent.notification.twilioApiKey" type="text" class="form-control">
-        <div class="form-text">
-            <p>
-                The API key is optional but recommended. You can provide either Account SID and AuthToken
-                from the may TwilioConsole page or Account SID and the pair of Api Key and Api Key secret
-            </p>
-        </div>
+        <div class="form-text">{{ $t("twilioApiKeyHelptext") }}</div>
     </div>
 
     <div class="mb-3">
-        <label for="twilio-auth-token" class="form-label">{{ $t("Auth Token / Api Key Secret") }}</label>
+        <label for="twilio-messaging-service-sid" class="form-label">{{ $t("twilioMessagingServiceSID") }}</label>
+        <input id="twilio-messaging-service-sid" v-model="$parent.notification.twilioMessagingServiceSID" type="text" class="form-control">
+        <i18n-t key="twilioMessagingServiceSIDHelptext" tag="div" class="form-text">
+            <template #twillo_messaging_service_help_link>
+                <a href="https://help.twilio.com/articles/223134387-What-is-a-Message-SID-" target="_blank">Twilio Messaging Service</a>
+            </template>
+        </i18n-t>
+
+    <div class="mb-3">
```

Comment:
fixed a linting mistake

```suggestion
        <i18n-t key="twilioMessagingServiceSIDHelptext" tag="div" class="form-text">
            <template #twillo_messaging_service_help_link>
                <a href="https://help.twilio.com/articles/223134387-What-is-a-Message-SID-" target="_blank">Twilio Messaging Service</a>
            </template>
        </i18n-t>
    </div>

    <div class="mb-3">
```